### ISSUE TITLE
Docs: `no-undef`: ESLint no longer supports jshint/jslint environments

### ISSUE THREAD

**Issue by IanVS** [eslint/eslint#3616]
The docs for [`no-undef`](http://eslint.org/docs/rules/no-undef#environments) talk about using `/*jslint browser:true*/` and `/*jshint node:true*/`.  I tried the example given, and it no longer works:

```js
/*jshint node:true*/
/*eslint no-undef: 2*/

var fs = require("fs");
module.exports = function() {
    console.log(fs);
};
```

gives:

```
  4:10  error  "require" is not defined  no-undef
  5:1   error  "module" is not defined   no-undef
  6:5   error  "console" is not defined  no-undef
```

I think  the docs for this rule should probably be updated to use `/*eslint-env*/` instead, yes?

**Comment by ilyavolodin** [comment]
I think the docs should be update in any case. I think support of jshint and jslint comments were removed a long time ago (somebody correct me if I'm wrong, it's been a while).

**Comment by nzakas** [comment]
Yup
